packages/experiments-realm/productivity/date-range-field.gts

```diff
@@ -0,0 +1,120 @@
+import DateField from 'https://cardstack.com/base/date';
+import {
+  FieldDef,
+  contains,
+  field,
+  Component,
+} from 'https://cardstack.com/base/card-api';
+import {
+  DateRangePicker,
+  BoxelDropdown,
+  Pill,
+} from '@cardstack/boxel-ui/components';
+import StringField from 'https://cardstack.com/base/string';
+import { action } from '@ember/object';
+import { tracked } from '@glimmer/tracking';
+
+const Format = new Intl.DateTimeFormat('en-US', {
+  year: 'numeric',
+  month: 'short',
+  day: 'numeric',
+});
+
+interface DateRange {
+  start: Date | null | undefined;
+  end: Date | null | undefined;
+}
+
+class Edit extends Component<typeof DateRangeField> {
+  @tracked range: DateRange = {
+    start: this.args.model.start,
+    end: this.args.model.end,
+  };
+
+  get formatted() {
+    if (!this.range.start && !this.range.end) {
+      return '[no date range]';
+    }
+    let start = this.range.start
+      ? Format.format(this.range.start)
+      : '[no start]';
```

Comment:
Not sure if this is necessary, as we will have a start date whenever the user clicks a date. else will be "no date range"